layouts/JsDocsLayout.re

```diff
@@ -236,10 +236,11 @@ module Docs = {
       theme=`Js
       components
       sidebar
-      breadcrumbs=?breadcrumbs
+      ?breadcrumbs
       route={
         router##route;
       }>
+      <Meta title={moduleName ++ " - ReasonML"} />
```

Comment:
No 100% sure about if this is the correct place to call this component, I would prefer to do:
```js
<>
 <Meta />
 ...rest of components
<>
```